manifests/kustomize/base/pipeline/ml-pipeline-ui-role.yaml

```diff
@@ -8,12 +8,9 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - pods
```

Comment:
these were previously needed when we didn't have viewer controller. Now kfp ui no longer need these permissions.